examples/incompressible-flow/2d-flow-around-cylinder/cylinder.prm

```diff
@@ -157,12 +156,8 @@ subsection linear solver
   subsection fluid dynamics
     set verbosity                             = verbose
     set method                                = gmres
-    set max iters                             = 5000
     set relative residual                     = 1e-4
     set minimum residual                      = 1e-9
-    set preconditioner                        = ilu
-    set ilu preconditioner fill               = 1
-    set ilu preconditioner absolute tolerance = 1e-12
-    set ilu preconditioner relative tolerance = 1.00
+    set preconditioner                        = amg
```

Comment:
Is there a reason why we use amg rather than ilu here?

Reply:
Yes, it's much faster if we use AMG, convergence is tremendously more robust.